ansible/catalog-web.yml

```
@@ -77,6 +77,57 @@
       when: datagov_in_service | default(true)
       tags:
         - smoke
+      
+    - name: check /_tracking first redirection
+      block:
+        - name: send POST
+          uri:
+            url: "{{ url_readonly }}/_tracking"
```

I think this makes more sense at the host level, not the service level.

```suggestion
            url: "{{ ansible_fqdn }}/_tracking"
```

---

Yes, I just thought this was getting tripped up on the cert and tried the more public URL. Now that I know about #900, I will roll back to this and utilize the `validate_certs` field in the uri module...